scripts/init-puppet.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+curl -k https://puppet.jenkins.io:8140/packages/current/install.bash | sudo bash
```

Comment:
just out of my own curiosity, I thought cloud_data scripts were executed as root already.

Reply:
Good point